app/database/models/server/category.ts

```diff
@@ -32,10 +32,10 @@ export default class CategoryModel extends Model implements CategoryInterface {
     /** associations : Describes every relationship to this table. */
     static associations: Associations = {
 
-        /** A CATEGORY has a 1:N relationship with CHANNEL. A CATEGORY can possess multiple channels */
+        /** A CATEGORY has a 1:N relationship with CATEGORY_CHANNEL. A CATEGORY can possess multiple channels */
         [CATEGORY_CHANNEL]: {type: 'has_many', foreignKey: 'category_id'},
 
-        /** A TEAM can be associated to CATEGORY (relationship is 1:N) */
+        /** A TEAM can be associated to multiple CATEGORY (relationship is 1:N) */
```

Comment:
I would also reword this. Maybe something like:
`A Category belongs to a Team, and a Team can have several categories (relationship 1:N)`